asterisk/rootfs/etc/asterisk/modules.conf

```diff
@@ -1,8 +1,5 @@
 [modules]
 autoload=yes
-noload => chan_sip.so
-noload => chan_alsa.so
-noload => chan_console.so
```

Comment:
I know nothing about this, so... if it's okay for you and @TECH7Fox, I'm good too.

Reply:
ihave disabled those modules during build. We do not have any sound output using ALSA HW.
Same goes for chan_console which uses portaudio to output directly to HW.
chan_sip will be disbaled during build with this PR.